en/companion_computer/README.md

```diff
@@ -14,14 +14,14 @@ They are connected using a fast serial or Ethernet link, and typically communica
 
 Communications with the ground stations and the cloud are usually routed via the companion computer (e.g. using the [MAVLink Router](https://github.com/mavlink-router/mavlink-router)).
 
-## Integrated Companion/Flight Controller Boards
+## Pixhawk Autopilot Bus Carrier Boards
 
-Controller boards that come with a pre-integrated companion computer and flight controller can significantly ease both software and hardware setup.
-In some cases the boards are set up to allow easy replacement of flight controller and/or companion computer parts.
+Carrier boards implementing the Pixhawk Autopilot Bus Open Standard with integrated support for advanced companion computer options.
```

Comment:
We should still mention PAB though - but link to https://docs.px4.io/main/en/flight_controller/pixhawk_autopilot_bus.html